packages/networks/src/networks/pulsechain.ts

```diff
@@ -0,0 +1,49 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const pulsechain: NetworkConfig = {
+    chain: 'pulsechain',
+    description: 'pulsechain mainnet.',
+    explorer: {
+        name: 'Pulsechain',
+        urls: {
+            address: (address: string) =>
+                `https://scan.pulsechain.com/address/${address}`,
+            base: `https://scan.pulsechain.com/`,
+            token: (address: string, holder: string) =>
+                `https://scan.pulsechain.com/token/${address}?a=${holder}`,
+            transaction: (hash: string) => `https://scan.pulsechain.com/tx/${hash}`,
+        },
+    },
+    featured: true,
+    fullySubsidizedGas: false,
+    id: 369,
+    isTestNetwork: false,
+    keyManagerAddress: '',
+    maxFreeClaimCost: 10000,
+    multisig: '',

```

Comment:
TK

IT looks like Gnosis Safe does not support this chain... can you please confirm?

Reply:
Correct they have not included pulsechain on the UI, however this network is one of the biggest users of gnosis safes via  
https://staker.app/ which is the UI used on pulsechain and ethereum. 

The liquid loans team have deployed a cut down version of gnosis on both testnet and mainnet at the below URLs.
https://pulsechainsafe.com/
https://testnet.pulsechainsafe.com/

Let me know if you need more information on these.

Reply:
At this point we don't want to add more work to our team to maintain non standards setup.